### ISSUE TITLE
ipp server exstension error.

### ISSUE THREAD

**Issue by drforbin** [sagemath/sage#38724]
### Steps To Reproduce

Just run sage -n

### Expected Behavior

Sage to run

### Actual Behavior

When the nodepad comes up I get apopup error of
ipp server extension error

### Environment

- **OS**: 
- **Sage Version**: 

### Checklist

- [X] I have searched the existing issues for a bug report that matches the one I want to file, without success.
- [X] I have read the documentation and troubleshoot guide

**Comment by drforbin** [comment]
Here is additional information



 ipyparallel | error adding extension (enabled: True): The module 'ipyparallel' could not be found (No module named 'ipyparallel'). Are you sure the extension is installed?
    Traceback (most recent call last):
      File "/home/mercou/sage-10.4/local/var/lib/sage/venv-python3.11/lib/python3.11/site-packages/jupyter_server/extension/manager.py", line 319, in add_extension
        extpkg = ExtensionPackage(name=extension_name, enabled=enabled)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/home/mercou/sage-10.4/local/var/lib/sage/venv-python3.11/lib/python3.11/site-packages/jupyter_server/extension/manager.py", line 183, in __init__
        self._load_metadata()
      File "/home/mercou/sage-10.4/local/var/lib/sage/venv-python3.11/lib/python3.11/site-packages/jupyter_server/extension/manager.py", line 198, in _load_metadata
        raise ExtensionModuleNotFound(msg) from None


It seems ipyparallel is not being loaded, for why I donot know.

**Comment by drforbin** [comment]
It was as simple as adding the path to site-packages to PYTHONPATH.
So case closed. Hope this helps someone else.

**Comment by drforbin** [comment]
Thxs everyone!
SAGE is great

**Comment by gwhitney** [comment]
In case it helps anyone else landing here, I just completed a build from source of a current clone of the sage git repository (sage 10.7.beta8, Release Date: 2025-07-06) on an OpenSUSE Linux installation, and when it was complete the sage REPL worked fine and passed all tests, but "sage -n" for the browser notebook interface repeatedly failed with symptoms just as described here.

However, in my case, the solution did not involve PYTHONPATH, it was to execute:

sage --python3 -m pip install ipyparallel

After this, the notebook appears to operate normally.

(This seems like a bug in the build process, that I needed to do this ipyparallel install manually. Should I file a separate issue for that?)
